torchvision/prototype/transforms/_meta.py

```diff
@@ -28,9 +28,11 @@ def __init__(self, dtype: torch.dtype = torch.float32) -> None:
         super().__init__()
         self.dtype = dtype
 
-    def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
+    def _transform(
+        self, inpt: Union[torch.Tensor, features.Image], params: Dict[str, Any]
+    ) -> Union[torch.Tensor, features.Image]:
         output = F.convert_image_dtype(inpt, dtype=self.dtype)
-        return output if features.is_simple_tensor(inpt) else features.Image.new_like(inpt, output, dtype=self.dtype)
+        return output if features.is_simple_tensor(inpt) else features.Image.new_like(inpt, output, dtype=self.dtype)  # type: ignore[arg-type]
```

Comment:
Mypy really couldn't understand that this can be a union.

Reply:
Not sure what the error is, but `Union[torch.Tensor, features.Image]` is redundant. Basically this says "the output can be a `torch.Tensor` **or** a `features.Image`". But since a `features.Image` is a `torch.Tensor`, `Union[torch.Tensor, features.Image]` should be equivalent to simply annotating with `torch.Tensor`.

Reply:
I believe earlier you introduced the usage of `DType = Union[torch.Tensor, PIL.Image.Image, features._Feature]`, which is also redundant for exactly the same reason.